packages/forms-web-app/src/controllers/examination/your-name.js

```diff
@@ -1,15 +1,46 @@
-const setData = () => {
-	return {
-		backLinkUrl: '#main-content'
-	};
-};
+const {
+	VIEW: {
+		EXAMINATION: { WHO_ARE_YOU_SUBMITTING_FOR, YOUR_NAME, YOUR_EMAIL_ADDRESS, ROUTE_PREFIX }
+	}
+} = require('../../lib/views');
+
+const setData = () => ({
+	backLinkUrl: WHO_ARE_YOU_SUBMITTING_FOR,
+	fullName: '',
+	title: 'What is your full name?'
+});
+
+const getYourName = async (req, res) => {
+	const { mySelfRegdata } = req.session;
+
```

Comment:
@mauriciokin Do we need to store the data in different keys depending on the users route? For example, if we used one key, like 'examination' for example (`const { examination } = req.session;`) we could then define full name with the same key for every journey. We can use the value of 'who are you submitting for' for the logic.

Reply:
Done